src/main/java/org/broadinstitute/hellbender/utils/R/RScriptExecutor.java

```diff
@@ -81,20 +81,19 @@ public RScriptExecutorException getScriptException(final String message) {
     }
 
     public boolean exec() {
-        List<File> tempFiles = new ArrayList<>();
+        final List<File> tempDirs = new ArrayList<>();
         try {
-            File tempLibSourceDir  = IOUtils.tempDir("RlibSources.", "");
-            File tempLibInstallationDir = IOUtils.tempDir("Rlib.", "");
```

Comment:
For consistency with `BaseTest.createTempDir()`, I think that `IOUtils.tempDir()` should register the temp dir for recursive deletion on exit via `IOUtils.deleteRecursivelyOnExit()`

Reply:
Done.